src/polyfill.ts

```diff
@@ -56,6 +127,11 @@ export function webkit(opts?: number | (PolyfillOption & { zoom?: number })) {
       value: true,
     })
 
+    // Repaint viewport forcibly when initial observing, to clear buggy SVG debris
+    document.body.style['zoom'] = 1.0001
+    void document.body.offsetHeight
+    document.body.style['zoom'] = 1
```

Comment:
If non-polyfilled SVG had been rendered before applying polyfills, WebKit sometime leaves SVG debris. An updated feature detection is asynchronous so we have to deal with debris.

This snippet triggers repaint in the whole of HTML contents to clear SVG debris in everywhere.